packages/knip/test/plugins/convex.test.ts

```diff
@@ -0,0 +1,24 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/convex');
+
+test('Find dependencies with the convex plugin', async () => {
+  const { /* issues, */ counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  // console.log({ issues });
+
+  assert.deepEqual(counters, {
+    ...baseCounters,
+    devDependencies: 1,
```

Comment:
If we `import "convex"` somewhere in one of our fixture files then we can clean up this test too (i.e. no more unused `devDependencies`)

Reply:
Done!